controllers/workspace/predicates.go

```diff
@@ -112,3 +126,12 @@ func objectIsAutomountResource(obj client.Object) bool {
 	}
 
 }
+
+func objectIsCertificateConfigmap(obj client.Object) bool {
+	configmapRef := config.GetGlobalConfig().Routing.TLSCertificateConfigmapRef
```

Comment:
Please check ` config.GetGlobalConfig().Routing` is not null as well

Reply:
done